cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/DatastoreImpl.java

```diff
@@ -1432,15 +1432,14 @@ private long insertDocumentHistoryToNewTree(SQLDatabase db, DocumentRevision new
         callable.available = !newRevision.isDeleted();
         long newLeafSeq = callable.call(db);
 
-        pickWinnerOfConflicts(db, docNumericID, newRevision.getId(), previousLeafSeq);
+        pickWinnerOfConflicts(db, docNumericID, newRevision.getId());
         return newLeafSeq;
     }
 
 
     private void pickWinnerOfConflicts(SQLDatabase db,
                                        long docNumericId,
-                                       String docId,
-                                       long previousWinnerSeq) throws DatastoreException {
+                                       String docId) throws DatastoreException {
```

Comment:
I can't see where in this method we use the `String docId` any more (or indeed where it was used before this change). Can we get rid of it?